1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,6 +26,15 @@ const AGES = [
     49
 ];
 
+function showWritersAge(name, age) {
+    for (let i = 0; i < name.length; i++) {
+        console.log(`${name[i]} is ${age[i]} years old.`);
+    }
+    return;
+}
+
+showWritersAge(WRITERS, AGES);
```

Comment:
Nice use of encapsulation and auxiliary functions!

Reply:
It's the fact that you have used the `showWritersAge` function to encapsulate your code, rather than just using a for loop 👍 
I have just noticed the redundant `return;` statement, it should be removed :)